pkg/cloud/services/ec2/eips.go

```diff
@@ -119,6 +119,7 @@ func (s *Service) releaseAddresses() error {
 	}
 
 	for _, ip := range out.Addresses {
```

Comment:
nit: Instead of shadowing the variable here, what about:
```go
for i := range out.Addresses {
	ip := out.Addresses[i]
}
```

Reply:
Yeah, I agree. Will make that change.

Reply:
Thanks, do you mind updating the other instances as well?

Reply:
All done. I think I got everything.